terraform/00_state.tf

```diff
@@ -26,6 +26,6 @@ terraform {
 data "terraform_remote_state" "state" {
     backend = "gcs"
     config = {
-        bucket = "${var.bucket_name}"
+        bucket = var.bucket_name
```

Comment:
Are all of these quote changes on purpose? What's the difference?

Reply:
To remove the warnings. Quotes are deprecated.

Reply:
ok sounds good